addons/godot-xr-tools/assets/hand.gd

```diff
@@ -20,15 +20,19 @@ signal hand_scale_changed(scale)
 
 
 # Last world scale (for scaling hands)
-var _last_world_scale := 1.0
+var _last_world_scale : float = 1.0
 
 
 # Capture the initial transform
-onready var _transform := transform
+onready var _transform : Transform = transform
 
 
 # Called every frame. 'delta' is the elapsed time since the previous frame.
 func _process(_delta: float) -> void:
+	# Do not process if in the editor
```

Comment:
`_process` only runs in the editor for tool scripts. I guess we could add this for clarity just in case someone decides to turn this into a tool script in the future but otherwise it's overkill.

Reply:
I found a few scripts with editor_hint checks that weren't tool scripts and cleaned those up.